### PR TITLE
Fixing route destitation for ipv6 static network configuration flow

### PR DESCRIPTION
**Pull request by nshidlin**
/cc @yevgeny-shnaidman @empovit

### CODE REVIEW

File: discovery-infra/test_infra/tools/static_network.py [hunk 1]
```diff
@@ -99,4 +99,8 @@ def _prepare_dns_resolver(machine_cidrs):
 def _prepare_routes(primary_mac, machine_cidrs):
     network = ip_network(machine_cidrs[0])
     gw_ip = str(ip_address(network.network_address) + 1)
-    return {'config': [{'destination': '0.0.0.0/0', 'next-hop-address': gw_ip, 'next-hop-interface': primary_mac.replace(':', ''), 'table-id': 254}]}
+    if network.version == 6:
+        default_route = '::/0'
+    else:
+        default_route = '0.0.0.0/0'
```
Comment (RazRegev) [inline discovery-infra/test_infra/tools/static_network.py:105]:
```suggestion
    default_route = '::/0' if network.version == 6 else '0.0.0.0/0'
```

Comment (nshidlin) [reply discovery-infra/test_infra/tools/static_network.py]: changed

**Review by RazRegev — APPROVED** [review]

**Pull request merged**